1.21.5 Crash when enabling mod

Hi!  When I attempt to enable the mod by pressing the "Enable visibility for all blocks" keybind, the game crashes.  Interestingly, it does not crash if the mod is already enabled for some reason (though I think this is only when barriers are visible).

Environment:
- macOS
- Prism launcher
- Minecraft `1.21.5`
- Fabric `0.16.4`
- Fabric API `0.120.0+1.21.5`
- Visible Barriers `2.0.22`
- Latest Sodium/Lithium
- some other mods that don't appear to be causing the crash

Based on the crash log, the error appears to be that the floaty item renderer is trying to treat an Item Frame as a LivingEntity, at `xyz.amymialee.visiblebarriers.util.FloatyRenderer.renderItem(FloatyRenderer.java:46)`.


Crash log:
```
[16:54:43] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: Rendering entity in world
	at knot/net.minecraft.class_898.method_3954(class_898.java:184) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_898.method_62424(class_898.java:160) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_761.method_22977(class_761.java:958) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_761.method_62207(class_761.java:835) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_761.method_62214(class_761.java:626) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_9909.method_61910(class_9909.java:67) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_761.method_22710(class_761.java:551) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_757.method_3188(class_757.java:756) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_757.method_3192(class_757.java:517) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1523(class_310.java:1353) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1514(class_310.java:936) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:265) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.14.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.14.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.14.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
Caused by: java.lang.ClassCastException: class net.minecraft.class_1533 cannot be cast to class net.minecraft.class_1309 (net.minecraft.class_1533 and net.minecraft.class_1309 are in unnamed module of loader 'knot' @77e9807f)
	at knot/xyz.amymialee.visiblebarriers.util.FloatyRenderer.renderItem(FloatyRenderer.java:46) ~[visiblebarriers-1.21.5-2.0.22.jar:?]
	at knot/xyz.amymialee.visiblebarriers.util.FloatyRenderer.render(FloatyRenderer.java:30) ~[visiblebarriers-1.21.5-2.0.22.jar:?]
	at knot/net.minecraft.class_897.handler$fia000$visiblebarriers$visibleBarriers$renderHead(class_897.java:3056) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_897.method_3936(class_897.java) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_915.method_3994(class_915.java:56) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_915.method_3936(class_915.java:31) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_898.method_68834(class_898.java:203) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_898.method_3954(class_898.java:178) ~[client-intermediary.jar:?]
	... 17 more
```

--- a/src/main/java/xyz/amymialee/visiblebarriers/util/FloatyRenderer.java
+++ b/src/main/java/xyz/amymialee/visiblebarriers/util/FloatyRenderer.java
@@ -43,7 +43,7 @@ public class FloatyRenderer {
         state.standingEyeHeight = entity.getStandingEyeHeight();
         state.positionOffset = null;
         state.squaredDistanceToCamera = MinecraftClient.getInstance().cameraEntity.squaredDistanceTo(entity);
-        this.itemModelManager.clearAndUpdate(state.itemRenderState, stack, ItemDisplayContext.GROUND, entity.getWorld(), (LivingEntity) entity, entity.getId());
+        this.itemModelManager.clearAndUpdate(state.itemRenderState, stack, ItemDisplayContext.GROUND, entity.getWorld(), entity instanceof LivingEntity living ? living : null, entity.getId());
         state.renderedAmount = 1;
         matrixStack.push();
         matrixStack.translate(0.0D, entity.getHeight() / 2, 0.0D);

